lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -1,3 +1,5 @@
+local wire_expression2_entity_trails = CreateConVar("wire_expression2_entity_trails", 50, FCVAR_ARCHIVE, "Amount of trails one E2 can make. (0 - to disable trails)", 0)
```

Comment:
The limit should be global across all created E2s.

Reply:
It is, just wrong description. Will fix tomorrow